shell/components/form/ClusterAppearance.vue

```diff
@@ -0,0 +1,125 @@
+
+<script>
+import ClusterIconMenu from '@shell/components/ClusterIconMenu';
+import { _CREATE, _VIEW } from '~/shell/config/query-params';
+import { CLUSTER_BADGE } from '@shell/config/labels-annotations';
+export default {
+  title:      'ClusterAppearance',
+  components: { ClusterIconMenu },
+  props:      {
+    name: { type: String, default: '' }, mode: { type: String, default: _CREATE }, currentCluster: { type: Object, default: null }
```

Comment:
Could you break this down into multiple lines? 

Just a personal opinion, but I think it makes "reading" the different variables much easier

```
props:      {
    name: { type: String, default: '' }, 
    mode: { type: String, default: _CREATE }, 
    currentCluster: { type: Object, default: null }
   ```